pipeline_live/data/sources/alpaca.py

```diff
@@ -49,9 +49,9 @@ def fetch(symbols):
                 if df.index.tzinfo is None:
                     df.index = df.index\
                         .tz_localize('UTC')\
-                        .tz_convert('US/Eastern')
+                        .tz_convert('US/Eastern').normalize()
                 else:
-                    df.index = df.index.tz_convert('US/Eastern')
+                    df.index = df.index.tz_convert('US/Eastern').normalize()
```

Comment:
🤔 Might not need to convert the timezone if we're also normalizing since normalizing will just send the time component to 00:00:00 regardless to timezone.

Reply:
ah that was my miss-understanding of normalize then, I'll remove it, only added it back due to it being present in the original but just forcing this to midnight sounds wrong